### PR TITLE
feat: display list of retrived documents when clicking on the chip

### PR DESCRIPTION
**Pull request by fontanierh**
pending discussion [here](https://github.com/dust-tt/decisions/issues/95)

Screenshot 2023-10-03 at 10 23 14

### CODE REVIEW

**Comment by spolu** [comment]
Can we try to align the results with the chip.
While you're there can you remove the X results when on mobile as it overflows if I remember correctly :pray:

**Comment by fontanierh** [comment]
> Can we try to align the results with the chip.

Will move to a grid layout @spolu  👍

**Comment by fontanierh** [comment]
@spolu addressed the alignment

Regarding the mobile thing, this is what it looks like rn:
Screenshot 2023-10-03 at 10 25 29


Think it's ~OK ? Would you rather not have the # of retrieved docs on small screens ?

**Comment by spolu** [comment]
Yes let’s remove on mobile as it’s non critical.

do we need an icon in the ship to show that it’s clickable?

File: front/components/assistant/conversation/RetrievalAction.tsx [hunk 1]
```diff
@@ -24,57 +26,104 @@ export default function RetrievalAction({
...
+              leaveTo="opacity-0 scale-95"
+            >
+              <ul className="ml-2 gap-2">
+                {retrievalAction.documents.map((document, i) => {
+                  const provider = providerFromDocument(document);
+                  return (
+                    <li key={i}>
+                      <a
+                        href={document.sourceUrl || ""}
+                        className="front-bold text-element-800"
```
Comment (spolu) [inline front/components/assistant/conversation/RetrievalAction.tsx:104]: shouldn't it be text-xs to be aligned with the text size of the retrieval component?

**Review by spolu — DISMISSED** [review]

**Comment by fontanierh** [comment]
Killed on phone + added a chevron icon 👍

**Pull request merged**